.github/workflows/test.yml

```diff
@@ -133,7 +133,7 @@ jobs:
       pull-requests: write # allows coverage bot to comment
     strategy:
       matrix:
-        python-version: ["3.9", "3.10", "3.11", "3.12", "3.13"]
+        python-version: ["3.9", "3.10", "3.11", "3.12"]
```

Comment:
We were never `3.13` compatible, because [qcs-sdk-python](https://pypi.org/project/qcs-sdk-python/) is not 3.13 compatible.

This stems from https://github.com/rigetti/rigetti-pyo3/issues/57